lib/charms/tls_certificates_interface/v1/tls_certificates.py

```diff
@@ -1279,10 +1279,10 @@ def _on_relation_changed(self, event: RelationChangedEvent) -> None:
         """
         relation = self.model.get_relation(self.relationship_name)
         if not relation:
-            logger.warning(f"No relation: {self.relationship_name}")
+            logger.debug(f"No relation: {self.relationship_name}")
```

Comment:
Is this a change of the lib?

Reply:
No, I patched it. I'm sick of the `update-status` warnings.

Reply:
I think it's changed in the V2+ versions of the lib.

I had raised an Issue for this a while ago, but apparently you can't release any more `V1` versions, once `V2` exists, so we're stuck with those noisy logs, or we patch.

Reply:
lol sounds good then